### PR TITLE
Display T proposals

### PR DESCRIPTION
**Pull request by r-czajkowski**
This PR adds support for fetching the latest proposal from https://forum.threshold.network via GraphQL query.

**GraphQL query**
```graphql
{
  allProposals(limit: 3) {
    title
    id
    createdBy
    createdAt
    url
    content {
      html
      raw
    }
  }
}
```

obraz

### CODE REVIEW

**Comment by georgeweiler** [comment]
Hey @r-czajkowski We discussed this today in the design sync. I think we want to pull the latest proposals, regardless of what they are.

File: gatsby/resolvers/proposal.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import axios from "axios"
+
+const BASE_API_URL = "https://forum.threshold.network"
+const PROPOSAL_CATEGORY_URL = `${BASE_API_URL}/c/governance-proposals/7.json`
```
Comment (georgeweiler) [inline gatsby/resolvers/proposal.ts:4]: just curious about this url. Why does it end in `/7`?

Comment (r-czajkowski) [reply gatsby/resolvers/proposal.ts:4]: It is the category id. Please see https://docs.discourse.org/#operation/listCategoryTopics

**Comment by georgeweiler** [comment]
@r-czajkowski this PR lgtm. Happy to merge if you agree.

**Comment by r-czajkowski** [comment]
> @r-czajkowski this PR lgtm. Happy to merge if you agree.

Your commits look good to me and thanks for the fix! Feel free to merge.

**Review by georgeweiler — APPROVED** [review]

**Pull request merged**